firestore/cloud-client/distributed_counters_test.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from google.cloud import firestore
+import distributed_counters
+
+
```

Comment:
```suggestion
shards_list = []
doc_ref = None

@pytext.fixture
def db_client():
      yield firestore.Client()
      
      # clean up
      for shard in shards_list:
          shard.delete()
      
      if doc_ref:
          doc_ref.delete()
```

Reply:
This ensures that the test properly cleans up even if fails

Reply:
Hm, on fail? That's good point